src/test/scala/ai/privado/languageEngine/python/passes/config/PythonPropertiesFilePassTest.scala

```diff
@@ -0,0 +1,57 @@
+package ai.privado.languageEngine.python.passes.config
+
+import ai.privado.languageEngine.java.passes.config.PropertiesFilePassTestBase
+import ai.privado.languageEngine.java.language._
+import io.shiftleft.semanticcpg.language._
+import io.shiftleft.codepropertygraph.generated.nodes.{JavaProperty, Literal}
+
+class GetEnvironmentTest extends PropertiesFilePassTestBase(".env", language = "python") {
+  override val configFileContents: String =
+    """
+       |MONGO_URL=mongodb+srv://myuser:mypassword@mycluster.abc123.mongodb.net/mydatabase?retryWrites=true&w=majority
+       |DEV=False
+       |""".stripMargin
+  override val codeFileContents: String =
+    """
+      |import os
+      |
+      |class Config:
+      |   url = environ.get("MONGO_URL")
+      |   db = MongoClient(url)
+      |""".stripMargin
+  override val propertyFileContents: String = ""
+
+  "ConfigFilePass" should {
+    "create a file node for the property file" in {
+      val List(name: String) = cpg.file.name.l
+      name.endsWith("/test.env") shouldBe true
+    }
+
+    "create a `property` node for each property" in {
+      val properties = cpg.property.map(x => (x.name, x.value)).toMap
+      properties
+        .get("MONGO_URL")
+        .contains(
+          "mongodb+srv://myuser:mypassword@mycluster.abc123.mongodb.net/mydatabase?retryWrites=true&w=majority"
```

Comment:
I see this String getting repeated many places, can we create a constant and refer that

Reply:
It is just a dummy string for test cases, do we really need to add it to the constants?

Reply:
Okay